app/assets/stylesheets/header.css.scss

```diff
@@ -0,0 +1,116 @@
+header {
+  min-height: 50px;
+}
+
+h1 {
+  color: $dark-blue;
+  font-size: 3em;
+  font-weight: bold;
+  text-align: center;
+}
+
+h2 {
+  font-size: 2em;
+  font-style: italic;
+  font-weight: bold;
+  color: $dark-blue;
+}
+
+.navigation-bar {
+  background: transparentize($blue, 0.2);
+  border-style: ridge none;
+  min-height: 50px;
+  position: relative;
+}
+
+#nav {
+  font-weight: bold;
+  left: 30%;
+  list-style-type: none;
+  margin-top: 0;
+  position: absolute;
+  width: 70%;
+
+  > li {
+    display: inline-block;
+    padding: 15px;
+  }
+
+  a {
+    text-decoration: none;
+   
+    &:hover {
+      text-decoration: underline;
+    }
+  }
+}
+
+.dropdown {
+  background: #fff;
+  background: rgba(255, 255, 255, 0);
+  display: none;
+  list-style: none;
+  position: absolute;
+  padding-left: 0;
+
+  li{
+    padding: 5px 0;
+    
+    a {
+      white-space: nowrap;
+    }
+  }
+}
+
+li:hover ul {
+  background: transparentize($blue, 0.2);
+  display: block;
+  margin-top: 10px;
+  position: absolute;
+  width: 100px;
+}
+
+
+.logo {
+  border-style: outset;
+  border-width: thick;
+  height: 200px;
+  margin-left: 50px;
+  margin-top: 15px;
+  width: 250px;
+}
+
+#banner-container {
+  position: relative;
+}
+
+#banner {
+  overflow: hidden;
+  height: 250px;
+  background: white;
+}
+
+.banner-image {
+  height: 250px;
+  width: 669px;
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  margin: auto;
+  left: 0;
+  right: 0;
+}
+
+#gradient {
+  position: absolute;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Properties should be ordered bottom, height, left, margin, position, right, top, width